ci-operator/step-registry/smongiar/deploy-resources/smongiar-deploy-resources-commands.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function create_foo_project_and_policies()
+{
+  oc new-project smongiar-interop
+  oc label --overwrite ns ${1} pod-security.kubernetes.io/enforce=privileged
+  oc label --overwrite ns ${1} pod-security.kubernetes.io/enforce-version-
+  oc adm policy add-scc-to-user privileged -z default -n ${1}
+}
+
+#function create_foo_secrets()
+#{
+#  oc create -f ./quay.io-secret.yml
+#  rm -f quaycfg.json
+#  oc secrets link default quay.io --for=pull
+#  oc secrets link builder quay.io --for=pull
+#  oc secrets link deployer quay.io --for=pull
+#}
+
+function create_foo_configmaps()
+{
+  cat << EOF > settings.xml
```

Comment:
```suggestion
  cat << EOF > /tmp/settings.xml
```

Reply:
new PR ready with this update

Reply:
@smongiar can we run rehearsal job now or are you still working on any changes?